composer.json

```diff
@@ -17,7 +17,7 @@
     },
     "require": {
         "neos/neos-development-collection": "2.0.x-dev",
-        "neos/flow-development-collection": "@dev",
+        "neos/flow-development-collection": "3.0.x-dev",
```

Comment:
it's been like this since we created the distributions, so yeah definitely tested it.. can't rule out composer changed anything though if it doesn't work anymore..

it's actually needed for all of the `.x-dev` since the stability flag comes from the root composer file and cannot come from a dependency